testing/supercronic/APKBUILD

```diff
@@ -0,0 +1,41 @@
+# Contributor: Thomas Kienlen <t.kienlen@adhoc-gti.com>
+# Maintainer: Thomas Kienlen <t.kienlen@adhoc-gti.com>
+pkgname="supercronic"
+pkgver="0.1.9"
+pkgrel=0
+pkgdesc="Cron for containers"
+url="https://github.com/aptible/supercronic"
+arch="all"
+license="MIT"
+depends=""
+makedepends="go bash dep python"
+install=""
+subpackages=""
+source="${pkgname}-${pkgver}.tar.gz::https://github.com/aptible/$pkgname/archive/v${pkgver}.tar.gz"
```

Comment:
1) You don't have to use curly braces here
2) Please replace the $pkgname variable in the url with the actual name, that way it's easier to paste it into the browser :)